tests/PlancakeEmailParserTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+declare(strict_types = 1);
```

Comment:
As much as I love PHP 7 and its strict typing, technically this library still supports PHP5. I'm fine with dropping support for anything prior to PHP 5.6, but ideally I still need to be able to run the tests on PHP 5.6 for the foreseeable future. Could you therefore please remove this.